tests/foreman/api/test_errata.py

```diff
@@ -55,6 +54,8 @@
 
 CUSTOM_REPO_URL = FAKE_6_YUM_REPO
 CUSTOM_REPO_ERRATA_ID = FAKE_2_ERRATA_ID
+DISTRO_RHEL6 = settings.distro.image_el6
```

Comment:
Are you sure settings are already initialized at this point? Please, provide test results. Some time ago this change would definitely not work, but maybe things have changed with introducing robozilla and settings are initialized earlier, i'm not sure.